File: src/main/java/com/freya02/botcommands/api/commands/prefixed/annotations/Hidden.java
package com.freya02.botcommands.api.commands.prefixed.annotations;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/** Use this annotation if you want the command to be hidden from everyone but the owners, <b>the 'help' command will not display it</b>
 */
@Retention(RetentionPolicy.RUNTIME)
@Target({ElementType.TYPE, ElementType.METHOD})
public @interface Hidden {}

File: src/main/kotlin/io/github/freya022/botcommands/api/commands/prefixed/annotations/Hidden.kt
package io.github.freya022.botcommands.api.commands.prefixed.annotations

import io.github.freya022.botcommands.api.commands.prefixed.builder.TextCommandBuilder

/**
 * Hides a command from help content and execution.
 *
 * @see TextCommandBuilder.hidden DSL equivalent
 */
@Retention(AnnotationRetention.RUNTIME)
@Target(AnnotationTarget.CLASS, AnnotationTarget.FUNCTION)
annotation class Hidden  